tools/linter/adapters/mypy_linter.py

```diff
@@ -118,6 +118,10 @@ def check_mypy_installed(code: str) -> list[LintMessage]:
         ]
 
 
+def in_github_actions() -> bool:
+    return bool(os.getenv("GITHUB_ACTIONS"))
```

Comment:
I wonder if this works.  This is run inside Nova Linux job docker container, so the environment variable needs to be passed through here https://github.com/pytorch/test-infra/blob/main/.github/scripts/run_with_env_secrets.py#L67-L73

Reply:
Yeah, it's probably from that env file. Looking at hud the issue isn't reproing once this PR was merged